### PR TITLE
[kubernetes/hdfs] Makes it possible to set additional Environmental variables

### PR DESCRIPTION
**Pull request by ahmetkaftan**
Signed-off-by: Ahmet Kaftan <ahmet.kaftan@cloudnesil.com>

### CODE REVIEW

File: kubernetes/hdfs/values.yaml [hunk 1]
```diff
@@ -46,6 +46,7 @@ namenode:
       resources:
         requests:
           storage: 10Gi
+  env: {}
```
Comment (t92549) [inline kubernetes/hdfs/values.yaml:49]: Perhaps the variable could be renamed to something like `extraEnvs` to be a little more descriptive.

File: kubernetes/hdfs/values.yaml [hunk 2]
```diff
@@ -46,6 +46,7 @@ namenode:
       resources:
         requests:
           storage: 10Gi
```
Comment (t92549) [inline kubernetes/hdfs/values.yaml:48]: Perhaps add a comment explaining the format of how to use this (variable.name: value)

**Review by t92549 — CHANGES_REQUESTED** [review]

File: kubernetes/hdfs/values.yaml (hunk 2 above)
Comment (ahmetkaftan) [reply kubernetes/hdfs/values.yaml:48]: Added.

File: kubernetes/hdfs/values.yaml (hunk 1 above)
Comment (ahmetkaftan) [reply kubernetes/hdfs/values.yaml]: Variable name updated.

File: kubernetes/hdfs/values.yaml (hunk 2 above)
Comment (t92549) [reply kubernetes/hdfs/values.yaml:48]: Really sorry to be picky but maybe adding a comment above these examples saying: "Example usage:" might be good just to be really clear

Comment (ahmetkaftan) [reply kubernetes/hdfs/values.yaml:48]: No problem at all, comment added.

**Review by t92549 — APPROVED** [review]

**Pull request merged**